packages/driver/cross-origin-testing.md

```diff
@@ -0,0 +1,141 @@
+# Cross-origin Testing Technical Overview
+
+The goal of this document is to give a technical overview of the architecture behind the **cy.origin()** command, which enables cross-origin testing in Cypress.
+
+## Definitions
+
+See [Node.js’s URL doc](https://nodejs.org/api/url.html#url-strings-and-url-objects) for a handy breakdown of URL parts
+
+- **domain**: The top-level domain or super-domain of a URL. A hostname without the subdomain. (e.g. `example.com`)
```

Comment:
Technically, the top-level domain is just the `com` part. https://developer.mozilla.org/en-US/docs/Glossary/TLD

Reply:
Derp - if I'm defining things, I should probably define them correctly!